evm-contracts/test/v0.6/VRFD20.test.ts

```diff
@@ -0,0 +1,251 @@
+import { contract, setup, helpers, matchers } from '@chainlink/test-helpers'
+import { assert } from 'chai'
+import { ContractTransaction } from 'ethers'
+import { VRFD20Factory } from '../../ethers/v0.6/VRFD20Factory'
+import { VRFCoordinatorMockFactory } from '../../ethers/v0.6/VRFCoordinatorMockFactory'
+
+let roles: setup.Roles
+const provider = setup.provider()
+const linkTokenFactory = new contract.LinkTokenFactory()
+const vrfCoordinatorMockFactory = new VRFCoordinatorMockFactory()
+const vrfD20Factory = new VRFD20Factory()
+
+beforeAll(async () => {
+  const users = await setup.users(provider)
+
+  roles = users.roles
+})
+
+describe('VRFD20', () => {
+  const deposit = helpers.toWei('1')
+  const fee = helpers.toWei('0.1')
+  const keyHash = helpers.toBytes32String('keyHash')
+  const seed = 12345
+
+  const requestId =
+    '0x66f86cab16b057baa86d6171b59e4c356197fcebc0e2cd2a744fc2d2f4dacbfe'
+
+  let link: contract.Instance<contract.LinkTokenFactory>
+  let vrfCoordinator: contract.Instance<VRFCoordinatorMockFactory>
+  let vrfD20: contract.Instance<VRFD20Factory>
+
+  const deployment = setup.snapshot(provider, async () => {
+    link = await linkTokenFactory.connect(roles.defaultAccount).deploy()
+    vrfCoordinator = await vrfCoordinatorMockFactory
+      .connect(roles.defaultAccount)
+      .deploy(link.address)
+    vrfD20 = await vrfD20Factory
+      .connect(roles.defaultAccount)
+      .deploy(vrfCoordinator.address, link.address, keyHash, fee)
+    await link.transfer(vrfD20.address, deposit)
+  })
+
+  beforeEach(async () => {
+    await deployment()
+  })
+
+  it('has a limited public interface', () => {
+    matchers.publicAbi(vrfD20Factory, [
+      // Owned
+      'acceptOwnership',
+      'owner',
+      'transferOwnership',
+      //VRFConsumerBase
+      'nonces',
+      'rawFulfillRandomness',
+      // VRFD20
+      'keyHash',
+      'fee',
+      'rollInProgress',
+      'rollDice',
+      'setKeyHash',
+      'setFee',
+      'latestResult',
+      'getResult',
+    ])
+  })
+
+  describe('#getResult', () => {
+    it('reverts when a number too high is used', async () => {
+      await matchers.evmRevert(async () => {
+        await vrfD20.getResult(99)
+      })
+    })
+
+    it('gets a previous result', async () => {
+      const randomness = 6
+      const modResult = (randomness % 20) + 1
+      await vrfD20.rollDice(seed)
+      await vrfCoordinator.callBackWithRandomness(
+        requestId,
+        randomness,
+        vrfD20.address,
+      )
+      const response = await vrfD20.getResult(0)
+      assert.equal(response.toString(), modResult.toString())
+    })
+  })
+
+  describe('#latestResult', () => {
+    it('reverts when there are no results', async () => {
+      await matchers.evmRevert(async () => {
+        await vrfD20.latestResult()
+      })
```

Comment:
```suggestion
        await vrfD20.latestResult(),
        'Invalid result number',
      })
```
Definitely worth making sure reverts are happening for the reason you suggest.